pages/units/[faction]/[type].tsx

```diff
@@ -71,13 +72,7 @@ const Unit: NextPage<{ unit: UnitDTO; terms: TermsDTO }> = ({
               <td>{bacteria.name}</td>
               <td>
                 <div>{bacteria.description}</div>
-                <div
-                  dangerouslySetInnerHTML={{
-                    __html: bacteria.modifierData
-                      .map((modifier) => modifier.description)
-                      .join("<br />"),
```

Comment:
One last issue, it was changed to display each modifier in a new line instead of comma separations. I would suggest to compare the output on soc.gg with your changes. You can take a look at the vercel deployment too, which is set for each pull request.